exercises/practice/knapsack/tests/knapsack.rs

```diff
@@ -31,7 +31,7 @@ fn test_no_items() {
     let max_weight = 100;
     let items = vec![];
```

Comment:
```suggestion
    let items = [];
```
I think these `vec!` macros can go too.